modules/ROOT/pages/_partials/data-sync/server-realtime-updates.adoc

```diff
@@ -66,20 +67,18 @@ The `createSubscriptionServer` code:
 ** managing websocket connections
 ** delivering subscriptions on the server
 * provides integrations with other modules such as `@aerogear/voyager-keycloak`.
```

Comment:
`.` at the end of this, but not for the previous two bullet points. Also, noticed that the previous two bullet points have 2 ** where this line only has *.